### ISSUE TITLE
Renaming file not allowed?

### ISSUE THREAD

**Issue by cheffNFT** [cardano-foundation/cardano-token-registry#942]
I can't rename a previously validated subject?

[Error]   [Cardano.Metadata.Validation.GitHub.validatePRFile#195] Pull request tried to rename file 'mappings/f10b16b705ef9baac04ca79c0da011633c8c20356b6a42262a1691da2d2043484546460a.json', but renaming entries is not permitted.
92

What should I do? create another one?

**Comment by d-sfounis** [comment]
By manually editing entries in the JSON metadata file, the signature component becomes invalid as it is the output of those entries. Try to edit fields thought the `metadata-tool,` and always re-sign after making changes

**Comment by cheffNFT** [comment]
hi I need some help... my token is not considered registered yet but an older version of it. can you assist me on what should I do?
https://cardanoscan.io/token/f10b16b705ef9baac04ca79c0da011633c8c20356b6a42262a1691da4348454646

**Comment by cheffNFT** [comment]
I submited another pull request...

./token-metadata-creator validate pull-request/mappings/f10b16b705ef9baac04ca79c0da011633c8c20356b6a42262a1691da4348454646.json
[Error]   [Main.log#290] Failed to validate wallet metadata entry, error was: 'attestation verification failed for: name'.
Error: Process completed with exit code 123.


I have no idea what it means. I tried to read and understand what should I do and still can't do that without knowing exactly how to proceed
